lib/solveEquation/stepThrough.js

```diff
@@ -107,6 +132,48 @@ function stepThrough(leftNode, rightNode, comparator, debug=false) {
   return steps;
 }
 
+/*
+  Helper function that returns the roots and symbol of an input equation
+  that has roots to be found.
+  For every factor on the left hand side, solve a new equation that is factor = 0
+  for the symbol.
+  TODO: handle multiple variable solutions e.g (x + 2) (y + 2) = 0
+*/
+
+function getSolutionsAndSymbol (equation) {
+  // Gets all factors that aren't a constant
```

Comment:
I don't think this comment is super descriptive - the name of the function is actually more accurate

maybe call the variable `factorsWithSymbols`?